com.unity.cinemachine/Samples~/3D Samples/Impulse/Impulse Wave.unity

```diff
@@ -0,0 +1,758 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!29 &1
```

Comment:
Help text should be fixed to describe the canvas way of invoking the impulse, and impulse listener on the vcam should be mentioned